templates/monasca-reconfigure.j2

```diff
@@ -19,4 +19,5 @@
     {% if monasca_user_domain_name is defined %} --user_domain_name '{{monasca_user_domain_name}}' {% endif %} \
     {% if monasca_user_domain_id is defined %} --user_domain_id '{{monasca_user_domain_id}}' {% endif %} \
     {% if monasca_skip_detection_plugins is defined %} --skip_detection_plugins '{{monasca_skip_detection_plugins}}' {% endif %} \
+    {% if monasca_agent_statsd_aggregation_interval is defined %} --monasca_statsd_interval '{{monasca_agent_statsd_aggregation_interval}}' {% endif %} \
```

Comment:
I don't see this parameter in master branch `monasca-setup --help` - do I need a patch to support it?

Reply:
Hmm, it should be there, but you need this patch which merged upstream 13 days ago:

https://github.com/openstack/monasca-agent/blame/master/monasca_setup/main.py#L431

I guess you might be using the stackhpc branch? I just merged the patch there too:

https://github.com/stackhpc/monasca-agent/pull/2